### ISSUE TITLE
Dockerfile does not compile and is therefore not released for 2.3.0

### ISSUE THREAD

**Issue by andymsuk** [bbc/bbc-a11y#291]
## Summary

Dockerfile does not compile (and is therefore not released) for 2.3.0

Running `docker build -t bbca11y/bbc-a11y-docker:latest -t bbca11y/bbc-a11y-docker:2.3.0 .` fails

## Expected Behaviour

Dockerfile builds.

## Current Behaviour

It fails with 
```W: Failed to fetch http://deb.debian.org/debian/dists/jessie-updates/InRelease  Unable to find expected entry 'main/binary-amd64/Packages' in Release file (Wrong sources.list entry or malformed file)

E: Some index files failed to download. They have been ignored, or old ones used instead.
Fetched 10.1 MB in 9s (1112 kB/s)
The command '/bin/sh -c wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - &&     sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google-chrome.list' &&     apt-get update -y &&     apt-get install -y libgconf-2-4 google-chrome-stable xvfb' returned a non-zero code: 100
```

## Possible Solution
Use a non-broken version of the node Docker image, such as `node:8.16.0-jessie`

## Steps to Reproduce (for bugs)

Run `docker build -t bbca11y/bbc-a11y-docker:latest -t bbca11y/bbc-a11y-docker:2.3.0 .` fails
